### DISCUSSION TITLE
GitHub setup and working

### DISCUSSION THREAD

**Discussion by Spidious** [MUsurf/JellyRos2#6] [Polls]
Hello, To make sure that everyone has gotten setup and working please respond to this discussion. 

Criteria are:
 1.  You have cloned the repository `JellyRos2` to your local machine using `git clone git@github.com:MUsurf/JellyRos2.git`
     - _You will need to enter the folder using `cd JellyRos2`_ 

2. You have changed your branch away from `dev` by running the command `git checkout TempTestBranch`
    - _Learn about how to use [git branches](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches) using the documentation and `git branch --help`_

3. You have added your GitHub username and real name to the `newuser.txt` file in the base directory of the repository
    - By this I mean the file is located in the very first part of the folder you cloned down
    - You can edit this using a text editor

4. Finally, You have committed and pushed your changes
    - To sync your changes to the remote repository you need to do three things
        1. Stage
        2. Commit
        3. Push
    - STAGE: add your changes to the staged changes with `git add newuser.txt` (you will need to be in the same directory as the file)
        - Check that this worked by running `git status` and see if newuser.txt is green
    - COMMIT: create a commit with your staged changes by running `git commit -m "My First Commit" 
    - PUSH: push your changes to the remote repository using 'git push'

**Comment by Thisiscole** [comment]
ERROR: Permission to MUsurf/JellyRos2.git denied to Thisiscole.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

**Reply by gywn9081** [reply]
This means you do not have access to the specific repository or do not have an authentication key in your GitHub account.

**Comment by MichaelBergen05** [comment]
👍It's done
